### PR TITLE
docs(adyen/paypal): Updated Paypal with Adyen documentation to include CSP and Adyen documents links

### PR DESCRIPTION
**Pull request by Kagaminara**
Added / completed PayPal part to Adyen documentation.

https://deploy-preview-449--heuristic-almeida-1a1f35.netlify.app/docs/advanced/payments/adyen

### CODE REVIEW

File: docs/advanced/payments/adyen.mdx [hunk 1]
```diff
@@ -283,6 +283,16 @@ In your Adyen Customer Area under `Developers > Webhooks` click on the
   - HMAC Key (HEX Encoded): click generate and copy it to
     `FRONT_COMMERCE_ADYEN_HMAC_KEY` environment variable defined above
 
+### Enable Paypal payments
+
+If you want to enable PayPal payments when using the Magento2 Adyen module, you
+will need to:
```
Comment (dpobel) [inline docs/advanced/payments/adyen.mdx:289]:
```suggestion
To enable PayPal payments, you need to:
```

File: docs/advanced/payments/adyen.mdx [hunk 2]
```diff
@@ -432,8 +442,15 @@ checking the "Save for my next payment" checkbox.
 ### Enable Paypal payments
 
 If you want to enable PayPal payments when using the Magento2 Adyen module, you
-will need to enable Adyen's `Alternative Payment Methods` located in
-`Stores > Configuration > Sales > Payment Methods > Adyen Payments > Configure payment methods`.
+will need to:
```
Comment (dpobel) [inline docs/advanced/payments/adyen.mdx:445]: same here:

```
To enable PayPal payments, you need to: 
```

**Review by abailleulfc — APPROVED** [review]
Except the missing link to the preview page in the first comment, all is good ;)

**Comment by Kagaminara** [comment]
> Except the missing link to the preview page in the first comment, all is good ;)

Edited 😁

**Pull request merged**